vue/src/store/modules/workspaces/state.js

```diff
@@ -47,6 +47,12 @@ const STATE = {
       },
       attachConfig: {},
     },
+    invite: {
```

Comment:
@skavt We do not need `invite` object, because we do not have separate invite page. We can directly have `inviteModal: {show: false, users: []}`